src/core/textures/Texture.js

```diff
@@ -216,15 +224,17 @@ export default class Texture extends EventEmitter
     {
         if (this.baseTexture)
         {
-            if (destroyBase)
+            // remove BaseTexture stored under the texture name if destroyBase is true or
+            // is our base texture (in which case this is the event callback for 'destroy')
+            if ((destroyBase === true || destroyBase === this.baseTexture)
```

Comment:
So this is destroying itself when the base texture is destroyed? I don't like that very much, seems totally unexpected that killing a child object somewhere in my code would then also destroy some other object.

Destroying a texture doesn't destroy the sprite it is on.